src/ledctl.c

```diff
@@ -723,7 +737,7 @@ static status_t slot_verify_request(struct slot_request *slot_req)
 		return STATUS_INVALID_STATE;
 	}
 	if (!slot_req->get_slot_fn && !slot_req->set_slot_fn) {
-		log_error("The controller type %s doesn't support slot functionality.",
+		log_error("The controller type %d doesn't support slot functionality.",
```

Comment:
controller is a string so why %d?

Reply:
> controller is a string so why %d?

`slot_req->cntrl` is an enumerated type

```C
struct slot_request {
	/**
	 * Option given in the request.
	 */
	int chosen_opt;

	/**
	 * Name of the device.
	 */
	char device[PATH_MAX];

	/**
	 * Unique slot identifier.
	 */
	char slot[PATH_MAX];

	/**
	 * Type of the controller.
	 */
	enum cntrl_type cntrl;

	/**
	 * IBPI state.
	 */
	enum ibpi_pattern state;

	/**
	 * Pointer to the get slot function.
	 */
	get_slot_t get_slot_fn;

	/**
	 * Pointer to the set slot function.
	 */
	set_slot_t set_slot_fn;
};
```

Reply:
ping?

Reply:
ok, sorry it is fixed in next patches :)